modules/ROOT/pages/dw-core-functions-reduce.adoc

```diff
@@ -23,9 +23,9 @@ accumulator parameter, a null value is returned.
 
 [%header, cols="1,3"]
 |===
-| Name   | Description
+| Name | Description
 | `item` | Item in the input array. It provides the value to reduce. Can also be referenced as `&#36;`.
-| `acc` a| The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation.
+| `acc` a | The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation.
```

Comment:
```suggestion
| `acc` a | The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the `reduce` operation.
```

Reply:
TODO: CHECK if space after "a" messes up the table; probably does. Better to have table directive [%header, cols="1,3a"], with the a here instead of in the table itself.